P4Plugin/Source/P4Task.cpp

```diff
@@ -678,13 +698,84 @@ bool P4Task::HasUnicodeNeededError( VCSStatus status )
 	return StatusContains(status, "Unicode server permits only unicode enabled clients");
 }
 
+bool P4Task::HasServerFingerPrintError(VCSStatus status)
+{
+	return StatusContains(status, "The authenticity of");
+}
+
 void P4Task::EnableUTF8Mode()
 {
 	CharSetApi::CharSet cs = CharSetApi::UTF_8;
 	m_Client.SetTrans( cs, cs, cs, cs );
 	m_Client.SetCharset("utf8");
 }
 
+#if defined(_WIN32) || defined(_WINDOWS)
+#define WIN32_LEAN_AND_MEAN
+#include <windows.h>
+#undef WIN32_LEAN_AND_MEAN
+
+bool P4Task::ShowOKCancelDialogBox(const std::string& windowTitle, const std::string& message)
+{
+	int msgBoxRet = MessageBoxW(
```

Comment:
Since you don't do actual UTF8 conversion, could just use MessageBoxA and without std::wstring conversions.

Reply:
Using the non-wide version in 11da728